ios/Classes/SwiftBluetoothEnablePlugin.swift

```diff
@@ -1,41 +1,55 @@
 import Flutter
 import UIKit
 import CoreBluetooth
-import os.log
+// https://stackoverflow.com/questions/39450534/cbcentralmanager-ios10-and-ios9/39498464#39498464
+
 
 public class SwiftBluetoothEnablePlugin: NSObject, FlutterPlugin, CBCentralManagerDelegate {
     var centralManager: CBCentralManager!
-    var lastKnownState: CBManagerState!
+    var lastKnownState = ""
```

Comment:
What's the difference between not initializing `lastKnownState` and set it to `""`?

Reply:
'CBManagerState' is only available in iOS 10.0 or newer
So i avoid using it